client/stores/addon.js

```diff
@@ -0,0 +1,30 @@
+import { observable, computed, action, runInAction } from 'mobx'
+import axios from 'axios'
+
+class AddonStore {
+	constructor () {
+		// Fetch list of addons from website server
+		this.getAddons()
+	}
+
+	@observable addons = []
+
+	@computed get addonsSortedByMostRecent () {
+		return this.addons.sort((a, b) => {
+			if (a.lastUpdated > b.lastUpdated) return -1
+			else if (a.lastUpdated < b.lastUpdated) return 1
+
+			return 0
+		})
+	}
+
+	@action getAddons = async () => {
+		const addons = await axios.get('/api/addons').then(res => res.data.addons)
+
+		runInAction(() => {
+			this.addons = addons
+		})
+	}
+}
+
+export default new AddonStore()
```

Comment:
Super duper minor nitpick: Would prefer if it was called `addons` store (plural). User store is singular as it was only supposed to hold the client alone (may also change, it's not scoped out).